PNG2Gas: cannot be built

The png2gas tool cannot be built like the [tutorial](http://picongpu.readthedocs.io/en/latest/install/dependencies.html?highlight=png2gas#png2gas) says.
```
[ 50%] Building CXX object CMakeFiles/png2gas.dir/png2gas.cpp.o
/HOME/src/picongpu/src/tools/png2gas/png2gas.cpp:128:5: warning: identifier ‘nullptr’ is a keyword in C++11 [-Wc++0x-compat]
     MPI_Init(nullptr, nullptr);
     ^
/HOME/src/picongpu/src/tools/png2gas/png2gas.cpp: In function ‘int main(int, char**)’:
/HOME/src/picongpu/src/tools/png2gas/png2gas.cpp:128:14: error: ‘nullptr’ was not declared in this scope
     MPI_Init(nullptr, nullptr);
              ^
make[2]: *** [CMakeFiles/png2gas.dir/png2gas.cpp.o] Error 1
make[1]: *** [CMakeFiles/png2gas.dir/all] Error 2
```

Exchanging `nullptr`back to `NULL` fixed it but that is a backport ^^;

--- a/src/tools/png2gas/png2gas.cpp
+++ b/src/tools/png2gas/png2gas.cpp
@@ -125,7 +125,7 @@ int main(int argc, char **argv)
     if (!parseCmdLine(argc, argv, options))
         return -1;
 
-    MPI_Init(nullptr, nullptr);
+    MPI_Init(NULL, NULL);
 
     Dimensions data_size(options.dataSize);
     std::cout << "Creating density data with size " << data_size.toString() << std::endl;

